coral/src/app/features/approvals/acls/AclApprovals.tsx

```diff
@@ -318,24 +318,25 @@ function AclApprovals() {
       // Warning: Encountered two children with the same key, ``.
       headerName: "",
       type: "custom",
-      UNSAFE_render: ({ id }: AclRequestTableRow) => {
+      UNSAFE_render: ({ id, requestStatus }: AclRequestTableRow) => {
```

Comment:
I miss tests for "does not show button when request status is not CREATED" 🙏

Reply:
Added! [510a75b](https://github.com/aiven/klaw/pull/666/commits/510a75bda1f62de629a1b772357b5efde4271c43)